src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableActorContext.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Provides functionality for application code implementing an actor operation.
+    /// </summary>
+    public interface IDurableActorContext : IDeterministicExecutionContext
+    {
+        /// <summary>
+        /// Gets the class of the currently executing actor.
+        /// </summary>
+        string ActorClass { get; }
+
+        /// <summary>
+        /// Gets the key of the currently executing actor.
+        /// </summary>
+        string Key { get; }
+
+        /// <summary>
+        /// Gets an actor reference for the currently executing actor.
+        /// </summary>
+        ActorId Self { get; }
+
+        /// <summary>
+        /// Gets the name of the operation that was called.
+        /// </summary>
+        string OperationName { get; }
+
+        /// <summary>
+        /// A logger for logging information about this actor.
+        /// </summary>
+        ILogger Logger { get; }
```

Comment:
What is the scenario for this?

Reply:
As mentioned above, removing this for now.